tests/integration/ceph_mgr_test.go

```diff
@@ -78,51 +78,74 @@ func (suite *MgrSmokeSuite) SetupSuite() {
 	suite.namespace = "mgr-ns"
 
 	mgrTestCluster := TestCluster{
-		namespace:        suite.namespace,
-		storeType:        "bluestore",
-		storageClassName: "",
-		useHelm:          false,
-		usePVC:           false,
-		mons:             3,
-		rbdMirrorWorkers: 1,
-		rookCephCleanup:  true,
-		withoutOSDs:      true,
+		namespace:               suite.namespace,
+		storeType:               "bluestore",
+		storageClassName:        "",
+		useHelm:                 false,
+		usePVC:                  false,
+		mons:                    1,
+		rbdMirrorWorkers:        0,
+		rookCephCleanup:         true,
+		skipOSDCreation:         true,
+		minimalMatrixK8sVersion: smokeSuiteMinimalTestVersion,
+		rookVersion:             installer.VersionMaster,
+		cephVersion:             installer.MasterVersion,
 	}
 
-	suite.op, suite.k8sh = StartTestCluster(suite.T, smokeSuiteMinimalTestVersion, &mgrTestCluster, installer.VersionMaster, installer.MasterVersion)
-
-	suite.CLI = MgrCLI{k8sh: suite.k8sh, namespace: suite.namespace}
-
+	suite.cluster, suite.k8sh = StartTestCluster(suite.T, &mgrTestCluster)
+	suite.waitOrchestrator()
 }
 
 func (suite *MgrSmokeSuite) AfterTest(suiteName, testName string) {
-	suite.op.installer.CollectOperatorLog(suiteName, testName, installer.SystemNamespace(suite.namespace))
+	suite.cluster.installer.CollectOperatorLog(suiteName, testName, installer.SystemNamespace(suite.namespace))
 }
 
 func (suite *MgrSmokeSuite) TearDownSuite() {
-	suite.op.Teardown()
+	suite.cluster.Teardown()
 }
 
+func (suite *MgrSmokeSuite) execute(command []string) (error, string) {
+	orchCommand := append([]string{"orch"}, command...)
+	return suite.cluster.installer.Execute("ceph", orchCommand, suite.namespace)
+}
+
+func (suite *MgrSmokeSuite) waitOrchestrator() {
+	output := ""
+	for timeout := 0; timeout < 30; timeout++ {
+		err, output := suite.execute([]string{"status"})
+		if err == nil {
+			logger.Info("%+v", output)
+			logger.Info("Rook Toolbox ready to execute commands")
+			break
+		}
+		time.Sleep(2 * time.Second)
```

Comment:
Also here I would suggest logging the err with every iteration so we can see what the test is waiting for

Reply:
yes